migrate/release_notes.md

```diff
@@ -0,0 +1,37 @@
+# Kava 16 Release Candidate 1 (v0.26.0-alpha.0)
+
+For deployment to Kava Mainnet at height `xxxx` around 2024-04-17 15:00:00 UTC.
+
+[Click to view upgrade instructions](https://github.com/Kava-Labs/kava/blob/v0.26.0/migrate/v0_26/migrate.md)
```

Comment:
It's best to link the branch since we have to update the commit placeholder after tagging